LLM/app/utils/DB/faissDB.py

```diff
@@ -3,20 +3,38 @@
 import numpy as np
 import pickle
 import os
+from googletrans import Translator
+
+
+class TextTranslator:
+    def __init__(self):
+        self.translator = Translator()
+
+    def translate_text(self, text, dest_lang='en'):
+        return self.translator.translate(text, dest=dest_lang).text
+
+
+# Создаем экземпляр класса TextTranslator
+translator = TextTranslator()
 
 
 def initialize_model():
     return SentenceTransformer('sentence-transformers/all-mpnet-base-v2')
 
 
 def create_embeddings(model, data):
-    texts = [item["question"] + " " + item["answer"] for item in data]
-    return model.encode(texts)
+    texts = [translator.translate_text(item["question"] + " " + item["answer"]) for item in data]
```

Comment:
```suggestion
    texts = [translator.translate_text('{} {}'.format(item["question"], item["answer"])) for item in data]
```

@BaldiSlayer вот так выглядит

Reply:
классно, спасибо)